### PR TITLE
Use storybook v4.0.0 alpha.11

### PR DESCRIPTION
**Pull request by sareh**
Use Storybook v4.0.0 alpha.11
This includes a fix to Storybook that was needed to complete this PR https://github.com/bbc/simorgh/pull/202

* [ ] Part of https://github.com/bbc/simorgh/issues/134
* ~[ ] Tests added for new features~

* [ ] Test engineer approval

### CODE REVIEW

**Comment by sareh** [comment]
Storybook in Travis CI can't handle the `forEach` in the `.storybook/config.js`. Will need to fix this.

File: .storybook/webpack.config.js [hunk 1]
```diff
@@ -0,0 +1,22 @@
+const path = require('path');
+
+module.exports = (baseConfig, env, defaultConfig) => {
+  defaultConfig.resolve.extensions.push('.jsx');
```
Comment (jtart) [inline .storybook/webpack.config.js]: We should be immutable rather than mutating `defaultConfig`.

**Review by jtart — CHANGES_REQUESTED** [review]

**Comment by jtart** [comment]
Can use a standard for loop for storybook config. forEach is slow and we shouldn't be in a habit of using it (although it would be fine for this purpose).

File: .storybook/webpack.config.js (hunk 1 above)
Comment (sareh) [reply .storybook/webpack.config.js]: I was just going with the recommended way of extending the WebPack config in the Storybook docs https://storybook.js.org/configurations/custom-webpack-config/#full-control-mode--default I can change it to be immutable, though.

**Comment by sareh** [comment]
Have investigated using `babel-polyfill` in the `.storybook/webpack.config.js`, however this doesn't seem to address the problem.  

There's an issue for this error here https://github.com/storybooks/storybook/issues/3830 One of the maintainers said it might be fixed in this WIP PR https://github.com/storybooks/storybook/pull/3746 but this is not guaranteed.

**Comment by sareh** [comment]
I investigated using Storybook 3, the stable version. It works well on a separate application, with these devDependencies:
```
    "@storybook/addon-actions": "^3.4.8",
    "@storybook/addon-links": "^3.4.8",
    "@storybook/addons": "^3.4.8",
    "@storybook/react": "^3.4.8",
    "babel-core": "^6.26.3",
    "babel-runtime": "^6.26.0",
```
However, since our application is using Webpack@4 and Storybook@3 uses Webpack@3, this conflicts.  
We have to wait until there is a fix to the issues in the Storybook@4 alpha. 

I will close this PR and open a new PR when the fixes have been made to Storybook. (https://github.com/storybooks/storybook/pull/3746)

**Pull request closed without merging**